docs/articles/documentation/guides/basic-guides/run-tests.md

```diff
@@ -50,6 +50,7 @@ TestCafe also allows you to create a [configuration file](../../reference/config
 * [Specify the Report Format](#specify-the-report-format)
 * [Customize Screenshot and Video Settings](#customize-screenshot-and-video-settings)
 * [Run Tests Concurrently](#run-tests-concurrently)
+  * [Use Concurrency on Remote Devices](#use-concurrency-on-remote-devices)
 * [Stage the Tested App](#stage-the-tested-app)
 * [Provide a Proxy URL](#provide-a-proxy-url)
```

Comment:
1) `* [Provide a Proxy URL](#provide-a-proxy-url)`

Can you rewrite this subtitle without using "Provide"?

2) I'm unable to edit line 38. Can you change it as follows?

`* [Run Tests from Multiple Sources](#run-tests-from-multiple-sources)`

Reply:
Okay, thank you, changed `Provide a Proxy URL` to `Specify a Proxy URL`

Reply:
Thanks, Vasily.